.pylintrc

```diff
@@ -93,6 +93,8 @@ ignore-imports=yes
 [MESSAGES CONTROL]
 
 disable=abstract-method,
+        bad-super-call,
+        super-with-arguments,
```

Comment:
Why are you adding this?

Reply:
In between there was some error so i added , will remove now.